docpages/building/linux.md

```diff
@@ -37,27 +37,27 @@ Once installed to the `/usr/local` directory, you can make use of the library in
 g++ -std=c++17 mydppbot.cpp -o dppbot -ldpp
 ```
 
-If you are on **Arch Linux**, you will need to add `/usr/local` to your environment variables:
+If you are on **Arch Linux**:
 
+- You need to give proper permission to `libdpp.so` and `libdpp.so.10.0.29`
 ```bash
-export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
-export CPLUS_INCLUDE_PATH=/usr/local/include:$CPLUS_INCLUDE_PATH
+sudo chmod 755 /usr/local/lib/libdpp.so.10.0.29
 ```
 
-However, if you don't want to change your environment settings, you can use these flags to compile:
-
+- You need to specify the location for `libdpp.so` when compilling:
 ```bash
-g++ -std=c++17 -L/usr/local/lib -I/usr/local/include mydppbot.cpp -o dppbot -ldpp
+g++ -std=c++17 -I/usr/local/include mydppbot.cpp -o dppbot /usr/local/lib/libdpp.so -Wl,-rpath,/usr/local/lib
 ```
 
 The important flags in this command-line are:
```

Comment:
I've also just realised this no longer exists for the Non-Arch Linux compile command (line 37). Please add the section about important flags for that.

Reply:
![image](https://github.com/brainboxdotcc/DPP/assets/46899449/90e302ba-4b03-46b1-82cf-b7d802027eb2)

You need the important flags for the section about compiling, where you aren't talking about arch linux.

Reply:
You should have two, keep the one you currently have, just duplicate this section and put it where the image above shows. Then correct the text.

Reply:
Yep perfect, thank you!